next-app/src/app/tokens/page.tsx

```diff
@@ -13,7 +13,10 @@ import {
   getSortedRowModel,
   useReactTable,
 } from "@tanstack/react-table";
-
+import { useAccount, useReadContract } from "wagmi";
+import { abi } from "../../abi/Token.json";
+import { ethers } from "ethers";
+import { useWriteContract } from 'wagmi'
 import { Button } from "../../components/ui/button";
```

Comment:
@/components For Next.

In tsconfig an path as be configured :     
"paths": {
      "@/*": ["./src/*"]
    }

Reply:
Still not corrected